test/core.jl

```diff
@@ -3106,3 +3106,8 @@ end
 const DATE12003 = DateTime(1917,1,1)
 failure12003(dt=DATE12003) = Dates.year(dt)
 @test isa(failure12003(), Integer)
+
+# Test error checking in bitstype
+@test_throws ErrorException bitstype 0 SPJ001
```

Comment:
OK, that was simply the convention we'd used for decades where I used to work, initials followed by number to mark changes (which was used as a job number in Perforce to track code changes)